examples/SoundClassification/sketch.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ml5 Example
+Sound classification using SpeechCommands18w and p5.js
+This example uses a callback pattern to create the classifier
+=== */
+
+// Initialize a sound classifier method with SpeechCommands18w model. A callback needs to be passed.
+let classifier;
+// Options for the SpeechCommands18w model, the default probabilityThreshold is 0
+const options = { probabilityThreshold: 0.7 };
+// Two variable to hold the label and confidence of the result
+let label;
+let confidence;
+
+async function setup() {
+    classifier = await ml5.soundClassifier("SpeechCommands18w", options);
+    // Create 'label' and 'confidence' div to hold results
+
+    label = document.createElement("DIV");
+    label.textContent = "label ...";
+    confidence = document.createElement("DIV");
+    confidence.textContent = "Confidence ...";
+
+    document.body.appendChild(label);
+    document.body.appendChild(confidence);
+    // Classify the sound from microphone in real time
+    classifier.classify(gotResult);
+}
+setup();
+
+// A function to run when we get any errors and the results
+function gotResult(error, results) {
```

Comment:
Noting that in #23 I changed the pattern of "error first" callbacks and this should now work with the simplified:

```javascript
function gotResult(results) {
}
```

Error checking is now optional!

```javascript
function gotResult(results, error) {
}
```